theme/js/components/dataset/resource/resources.vue

```diff
@@ -1,13 +1,27 @@
 <template>
-  <h3 v-if="showTitle" class="fr-mt-4w fr-mb-1w fr-text--sm fr-text--bold text-transform-uppercase" ref="top">
-      {{ typeLabel }} <sup v-if="showTotal">{{ totalResults }}</sup>
-  </h3>
-  <section class="resources-wrapper" key="top">
+  <section class="resources-wrapper fr-grid-row fr-mb-3v" v-if="showTitle" ref="top">
+    <div class="fr-col-12 fr-grid-row">
+      <div class="fr-col">
+        <h2 :class="{'fr-mt-4w': !firstGroup}" class="fr-mb-0 subtitle subtitle--uppercase">
+          {{ typeLabel }}
+        </h2>
+      </div>
+      <div class="fr-col-auto" v-if="isCommunityResources">
+        <a class="fr-btn fr-btn--sm fr-btn--secondary fr-btn--secondary-grey-500 fr-icon-add-line fr-btn--icon-left"
+          :href="newResourceAdminPath"
+        >
+          {{ $t('Add a community resource') }}
+        </a>
+      </div>
+    </div>
+    <div class="fr-col-12" v-if="showSearch">
+      <SearchBar eventName="resources.search"></SearchBar>
```

Comment:
Shouldn't SearchBar be before the typeLable since if filters on all types?

Reply:
I'll update the js side to only filter the current type

Reply:
I'm using this as an opportunity to update our events types